### ISSUE TITLE
Notification light/LED not blinking

### ISSUE THREAD

**Issue by SeriousHoax** [k9mail/k-9#3160]
### Expected behavior
The led notification light should blink everytime an email arrives as I have the option enabled.

### Actual behavior
Led notification light doesn't blink at all. My device has only one colored notification light and that is white. In the color option, there is no option for selecting white too. Well, it should work anyway as it does on any other app, so I think the colored doesn't matter here. I have one Gmail and one Outlook account logged in. It's not working on any of them.

### Environment
K-9 Mail version: 5.403

Android version: 8.0.0 Device: Xiaomi Mi A1

Account type (IMAP, POP3, WebDAV/Exchange): I'm not aware of this. I just added my Gmail and Outlook account via the default method.

**Comment by elgrafico** [comment]
Can confirm for OnePlus 5T with Android 8.0.0 and before that, Android 7.? as well.

K9 v5.4.0.3 and v5.4.0.2

**Comment by AvoidMastodonBlue** [comment]
Instead I use the app: LightFlow

**Comment by Hawk777** [comment]
Do you have a ringtone enabled for your new mail notification? I just upgraded to Android 8 and K-9 Mail stopped flashing the LED for new mail. I had no ringtone enabled, just the LED. I tried enabling a ringtone, and that worked, and the LED flashed as well. I didn’t actually want a ringtone, so I just created an empty (zero-seconds-long) WAV file, installed it as a ringtone called Silence, and selected that in K-9 Mail account settings. It’s obviously a hacky workaround, not a proper solution, but it may (1) help you until the bug is fixed properly, and (2) provide some avenue of investigation for fixing the bug. Again, I don’t know if this is the same problem or a different problem, but if you did previously have only LED (not ringtone) configured, try it out; it might be the same problem.

**Comment by Diapolo** [comment]
I can confirm that after an upgrade to Oreo (LineageOS 15.1) on my FP2 K9 doesn't show any notification LED :-/. Will try to set a notification sound, but that shouldn't be necessary IMHO.

**Comment by Diapolo** [comment]
It's not only a notification light problem IMHO there are some serious issues with notifications in general on Oreo+. As an example I've got 3 accounts configured and for one I simply get zero notifications, even if configured exactly as my main account, which is working most of the time.

**Comment by IgitBuh** [comment]
I'm experiencing the same issue. The workaround works, but it has an annoying downside for me: If I set my phone to vibration mode, every audio notification is automatically turned into a vibration notification. This means, that by setting even an "empty" ringtone, the phone is vibrating with every new email.
I need to be in vibration mode most of the time, so currently I can only have no LED notification for K-9 emails or LED and vibration for every new email. :(

**Comment by cketti** [comment]
Duplicate of #3098

**Comment by Diapolo** [comment]
@cketti You are tagging this as duplicate and close the original thread also by telling this is "normal" Android behaviour? Really? Did you release an update fixing notification stuff?

All other apps I'm using sorted out their notification issues with oreo, just K9 did not until now... I have 3 accounts setup, one is not giving any LED indicator, a second is just doing it from time to time and the single account that works needs a sound for LED blinks, which is just nonsense. Why is e.g. Signal able to handle the notifications?
